tools/topology/topology2/cavs/cavs-passthrough-hda.conf

```diff
@@ -1,23 +1,38 @@
 Object.Dai {
-	HDA.0 {
+	HDA.1 {
```

Comment:
commit message: technically it's not FE and BE, but rather host/dai or host/link.

The reference to ASoC FE and BE is a bit misleading, there is no clear assignment of a mixer path to a BE. that's a gray area really.